maven-plugin-plugin/src/main/java/org/apache/maven/plugin/plugin/AbstractGeneratorMojo.java

```diff
@@ -313,7 +305,7 @@ static String getDefaultGoalPrefix( MavenProject project )
         Set<Artifact> filteredDependencies;
         if ( mojoDependencies == null )
         {
-            filteredDependencies = dependencies;
+            filteredDependencies = new LinkedHashSet<>( project.getArtifacts() );
```

Comment:
dependencies or artifacts?

Reply:
Renaming `filteredDependencies` to `filteredArtifacts` should be sufficient, right?

Reply:
done